o-clock.cabal

```diff
@@ -32,7 +32,7 @@ source-repository head
 
 common common-options
   build-depends:
-      base >=4.12 && <4.17
+      base >=4.12 && <4.18
```

Comment:
I think it should be safe to set base's upper bound to `< 5`. IME, most packages do that.